ckanext/canada/tables/briefingt.yaml

```diff
@@ -37,6 +40,7 @@ resources:
       en: This field is populated by the user with the official title of the Briefing Note, in English. (Will need guidance here for redactions/exceptions).
     obligation: Mandatory
     excel_required: true
+    form_required: true
     format_type: Free text
     validation: This field must not be empty
```

Comment:
please add form attributes for size, example: https://github.com/open-data/ckanext-canada/blob/a46856aed344c0e162d0b5f49909b498000edaeb/ckanext/canada/tables/travelq.yaml#L72